mappings/maestro_analysis_centric.json

```diff
@@ -97,6 +97,19 @@
       "files": {
         "type": "nested",
         "properties": {
+          "file_id": {
+            "type": "keyword",
+            "copy_to": ["analysis_autocomplete"]
+          },
+          "release_stage": {
+            "type": "keyword"
+          },
+          "program_access_date": {
+            "type": "date"
+          },
+          "publish_date": {
```

Comment:
the publish date will likely be part of the SONG Analysis body - does it make more sense for it to be listed in the analysis  object? https://github.com/icgc-argo/argo-metadata-schemas/blob/281687195cf763b222f314df9c78341ccfd508b3/mappings/maestro_file_centric.json#L82 

Same comment for the `file_centric` below as well.